### PR TITLE
CAMEL-15658: camel-pgevent : automate integration tests

### PR DESCRIPTION
**Pull request by zbendhiba**

### CODE REVIEW

File: components/camel-pgevent/pom.xml [hunk 1]
```diff
@@ -64,6 +65,94 @@
...
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-testcontainers-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
+    <profiles>
+        <profile>
+            <id>testcontainers-skip-tests</id>
```
Comment (oscerd) [inline components/camel-pgevent/pom.xml:78]: Maybe it's better to have the component name too here

File: components/camel-pgevent/pom.xml [hunk 2]
```diff
@@ -64,6 +65,94 @@
...
+                        <configuration>
+                            <skipTests>true</skipTests>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!-- activate integration test if the docker socket file is accessible -->
+        <profile>
+            <id>testcontainers-integration-tests-docker-file</id>
```
Comment (oscerd) [inline components/camel-pgevent/pom.xml:95]: ditto

File: components/camel-pgevent/pom.xml [hunk 3]
```diff
@@ -64,6 +65,94 @@
...
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <!-- activate integration test if the DOCKER_HOST env var is set -->
+        <profile>
+            <id>testcontainers-integration-tests-docker-env</id>
```
Comment (oscerd) [inline components/camel-pgevent/pom.xml:127]: ditto

**Review by oscerd — CHANGES_REQUESTED** [review]
Just minor findings.

File: components/camel-pgevent/pom.xml (hunk 1 above)
Comment (zbendhiba) [reply components/camel-pgevent/pom.xml:78]: OK

Comment (zbendhiba) [reply components/camel-pgevent/pom.xml]: done !!

File: components/camel-pgevent/pom.xml (hunk 2 above)
Comment (zbendhiba) [reply components/camel-pgevent/pom.xml]: done

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Pull request merged**